webpack.config.js

```diff
@@ -33,7 +33,7 @@ if (fileSystem.existsSync(secretsPath)) {
   alias['secrets'] = secretsPath;
 }
 
-var options = {
+let options = {
```

Comment:
Nitpick. This doesn't need to be a `let`. Setting it to `const` is fine, even though you're updating something within the object.